lib/i18n/backend/chain.rb

```diff
@@ -57,6 +57,31 @@ def translate(locale, key, default_options = EMPTY_HASH)
           namespace = nil
           options = default_options.except(:default)
 
+          if key.is_a?(Array)
+            results = Array.new(key.length) { |_| nil }
+            namespaces = Array.new(key.length) { |_| nil }
+            untranslated_indices = (0...key.length).to_a
+
+            backends.each do |backend|
+              break if untranslated_indices.empty?
```

Comment:
Seems like this should be an if statement and you shouldn't loop through `backends` if this is empty, instead of breaking each time immediately.